### PR TITLE
Docker metrics modules

### PR DESCRIPTION
**Pull request by imnotashrimp**
## Pages to review

- https://deploy-preview-264--logz-docs.netlify.com/shipping/metrics-sources/docker.html

**Notes for reviewers**:

Technical and copy review, please

## Remaining work

- [x] Technical review
- [x] Copy Review

## Post launch

**Do not remove** - To be completed by the docs team upon merge:
- [x] Teams to update with the new information: Ido
- [ ] convert readme for the original repo

### CODE REVIEW

File: _source/logzio_collections/_metrics-sources/docker.md [hunk 1]
```diff
@@ -38,7 +39,7 @@ shipping-tags:
     logzio/docker-collector-metrics
```
Comment (idohalevi) [inline _source/logzio_collections/_metrics-sources/docker.md]: Please add `LOGZIO_MODULES="docker"` to the sample code

File: _source/logzio_collections/_metrics-sources/docker.md [hunk 2]
```diff
@@ -50,26 +51,60 @@ shipping-tags:
...
+      for connecting to Docker over TLS.
+
+    DOCKER_KEY
+    : Filepath to Docker key
+      for connecting to Docker over TLS.
 
-    <div class="info-box note">
-      By default, logs from docker-collector-logs and docker-collector-metrics containers are ignored.
-    </div>
+    By default, logs from docker-collector-logs and docker-collector-metrics containers are ignored.
```
Comment (idohalevi) [inline _source/logzio_collections/_metrics-sources/docker.md]: `By default, logs from docker-collector-logs and docker-collector-metrics containers are ignored.`
Doesn't make any sense. We collect metrics, not logs

Comment (idohalevi) [reply _source/logzio_collections/_metrics-sources/docker.md]: Since it's a stand-alone page, maybe we can take this comment out?

Comment (imnotashrimp) [reply _source/logzio_collections/_metrics-sources/docker.md]: What about this instead? Looks like it was just an oversight.

```
By default, metrics from docker-collector-logs and docker-collector-metrics containers are ignored.
```

Comment (idohalevi) [reply _source/logzio_collections/_metrics-sources/docker.md]: It's not true for the metric. I believe it's true for logs only :) let's take it out

Comment (imnotashrimp) [reply _source/logzio_collections/_metrics-sources/docker.md]: And will it report metrics on itself?

Comment (idohalevi) [reply _source/logzio_collections/_metrics-sources/docker.md]: I believe it does

File: _source/logzio_collections/_metrics-sources/docker.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ logo:
   orientation: horizontal
 data-source: Docker
 open-source:
-  - title: docker-collector-metrics
+  - title: Docker Metrics Collector
```
Comment (imnotashrimp) [inline _source/logzio_collections/_metrics-sources/docker.md]:
```suggestion
  - title: Docker Collector (Metrics)
```

File: _source/logzio_collections/_metrics-sources/docker.md [hunk 4]
```diff
@@ -14,13 +14,14 @@ shipping-tags:
   - container
 ---
 
-## docker-collector-metrics setup
+## Docker Metrics Collector setup
```
Comment (imnotashrimp) [inline _source/logzio_collections/_metrics-sources/docker.md]:
```suggestion
## Docker Collector (Metrics) setup
```

File: _source/logzio_collections/_metrics-sources/docker.md [hunk 5]
```diff
@@ -14,13 +14,14 @@ shipping-tags:
...
 
 ###### Configuration
 
 1.  Pull the Docker image
 
-    Download the logzio/docker-collector-metrics image:
+
+    Download the Docker Metrics Collector image:
```
Comment (imnotashrimp) [inline _source/logzio_collections/_metrics-sources/docker.md]:
```suggestion
    Download the Docker Collector (Metrics) image:
```

File: _source/logzio_collections/_metrics-sources/docker.md (hunk 1 above)
Comment (imnotashrimp) [inline _source/logzio_collections/_metrics-sources/docker.md]:
```suggestion
    LOGZIO_MODULES="docker" \
    logzio/docker-collector-metrics
```

File: _source/logzio_collections/_metrics-sources/docker.md (hunk 2 above)
Comment (imnotashrimp) [inline _source/logzio_collections/_metrics-sources/docker.md]:
```suggestion
```

File: _source/logzio_collections/_metrics-sources/docker.md (hunk 1 above)
Comment (imnotashrimp) [reply _source/logzio_collections/_metrics-sources/docker.md]: Done.

**Comment by idohalevi** [comment]
@imnotashrimp Maybe we should change the title to docker metrics? or even just docker if it sits under the metrics tab?

**Comment by imnotashrimp** [comment]
> @imnotashrimp Maybe we should change the title to docker metrics? or even just docker if it sits under the metrics tab?

image

Do you mean the name on the card?

**Comment by idohalevi** [comment]
@imnotashrimp My mistake. I was sure the title is the name of the card. I feel we can merge

File: _source/logzio_collections/_metrics-sources/docker.md [hunk 6]
```diff
@@ -50,26 +49,59 @@ shipping-tags:
...
+      for connecting to Docker over TLS.
+
+    DOCKER_KEY
+    : Filepath to Docker key
+      for connecting to Docker over TLS.
 
-    <div class="info-box note">
-      By default, logs from docker-collector-logs and docker-collector-metrics containers are ignored.
-    </div>
+    {:.info-box.note}
```
Comment (imnotashrimp) [inline _source/logzio_collections/_metrics-sources/docker.md]:
```suggestion
```

**Pull request merged**